ByeAds.py

```diff
@@ -37,13 +37,15 @@ def restartSpotify():
     closeSpotify()
     openSpotify()
     playSpotify()
-
-
-spotifyUsername = "#EnterUsernameHere"
-spotifyClientID = "#EnterClientIDHere"
-spotifyClientSecret = "#EnterClientSecretHere"
-spotifyAccessScope = "user-read-currently-playing user-modify-playback-state"
-spotifyRedirectURI = "http://google.com/"
+#load options.json
+with open('options.json', 'r') as f:
+    options = json.load(f)
+#Set variables equal to options.json equivalent
+spotifyUsername = options["spotifyUsername"]
+spotifyClientID = options["#spotifyClientID"]
```

Comment:
```suggestion
spotifyClientID = options["spotifyClientID"]
```

Shouldn't it be like this? The hashtag isn't in the key name in the JSON file.

Reply:
Yes, sorry for the error

Reply:
You can accept my suggestion directly via GitHub.

Reply:
Sorry, I didn't know that. I just entered it myself.